lua/starfall/libs_sh/convar.lua

```diff
@@ -0,0 +1,129 @@
+
+local checkluatype = SF.CheckLuaType
+local registerprivilege = SF.Permissions.registerPrivilege
+local PLY_META = FindMetaTable("Player")
+
+if CLIENT then
+	registerprivilege("convar", "Read ConVars", "Allows Starfall to read your game settings", { client = {} })
+	registerprivilege("convar.userinfo", "Retrieves userinfo associated with client-side ConVar (with `getUserInfo`).", "Allows users to retrieve value of userinfo ConVars.", { client = { default = 1 } })
```

Comment:
I don't think a separate permission is needed.

Reply:
I guess that makes more sense, since `convar.getString` doesn't treat userinfo differently anyways...